ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -232,43 +206,40 @@ describe('ActivitiPeople', () => {
             jasmine.Ajax.uninstall();
         });
 
-        it('should log error message when search fails', async(() => {
+        it('should log error message when search fails', () => {
```

Comment:
i don't think the test is testing correctly the behaviour without the async clause here.